keras_hub/src/models/efficientnet/fusedmbconv.py

```diff
@@ -77,6 +77,7 @@ def __init__(
         activation="swish",
         dropout=0.2,
         nores=False,
```

Comment:
the docstring args are not matching the arg list here - please make sure they are consistent

Reply:
done